packages/components/managed-data-table/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from './managed-data-table';
+export { useRowSelection, useSorting } from '@commercetools-uikit/hooks';
```

Comment:
Why are you re-exporting hooks from another package?

Reply:
My understanding is that you do that in order to allow the independent component dependency instead of the whole UIKit. This way you can only add the `ManagedDataTable` and use the `userRowSelection` and `useSorting` hooks importing them as 
```
import { useRowSelection, useSorting } from '.';
```
on the parent component

Reply:
Sure, that's fine. I was asking in case there were other reasons. Thanks.